### PR TITLE
Release v0.7.3

### PR DESCRIPTION
**Pull request by damian5710**
### Changed
- use lxml parser instead of the default xml parser

### CODE REVIEW

File: mffpy/xml_files.py [hunk 1]
```diff
@@ -65,14 +65,17 @@ def register(typ, xml_type):
...
     @classmethod
-    def from_file(typ, filepointer: FilePointer):
+    def from_file(typ, filepointer: FilePointer, recover=True):
         """return new `XMLType` instance of the appropriate sub-class
 
         **Parameters**
         *filepointer*: str or IO[bytes]
             pointer to the xml file
+        *recover*: bool
+            indicates whether to try hard to parse through broken XML or not
```
Comment (ianbrown9475) [inline mffpy/xml_files.py:75]: Can you document here that we set this to true by default because it's necessary if there are weird characters in the xml file, which can occasionally occur?

File: mffpy/xml_files.py [hunk 2]
```diff
@@ -65,14 +65,17 @@ def register(typ, xml_type):
             return False
 
     @classmethod
-    def from_file(typ, filepointer: FilePointer):
+    def from_file(typ, filepointer: FilePointer, recover=True):
```
Comment (ianbrown9475) [inline mffpy/xml_files.py:68]: Can you annotate the type of `recover` as a bool?

**Review by ianbrown9475 — CHANGES_REQUESTED** [review]

**Review by ianbrown9475 — APPROVED** [review]

**Review by ephathaway — CHANGES_REQUESTED** [review]
Please update the test coverage in the README, then this should be good to go.

**Review by ephathaway — APPROVED** [review]
Thanks. Once it's merged, I can upload the release to pypi.

**Review by ianbrown9475 — APPROVED** [review]
